src/main/config/config.c

```diff
@@ -591,7 +591,11 @@ static void resetConf(void)
 #endif
 
     // Radio
+#ifdef RX_CHANNELS_TAER
```

Comment:
All AlienFlight targets would also be able to make use of this instead setting this in the target/config.c.